examples/jsm/loaders/KTX2Loader.js

```diff
@@ -864,18 +864,18 @@ async function createRawTexture( container ) {
 
 	if ( UNCOMPRESSED_FORMATS.has( FORMAT_MAP[ vkFormat ] ) ) {
 
+		const textureData = mipmaps.length > 0 ? mipmaps[ 0 ].data : null;
```

Comment:
The length of `mipmaps` must be at least 1, so it would be fine to use `mipmaps[ 0 ].data` unconditionally here.

Reply:
done!